apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -141,6 +152,18 @@ defmodule Aecore.Chain.Worker do
     get_blocks(top_block_hash(), top_height() + 1)
   end
 
+  def all_registered_voting_questions() do
```

Comment:
Can the function calls for voting (`all_registered_voting_questions`, `get_voting_question_by_hash` and `get_voting_result_for_a_question`) be moved to the voting manager, for a cleaner separation?

In the chain worker you can leave a `get_voting_state` and use this as a call in the voting manager.